git_commit_all_submodules.sh

```
@@ -0,0 +1,56 @@
+    fi
+    git add "$SUBMODULE";
+  fi
+done
+
+echo "$COMMITS change(s) to commit.";
+
+if [[ "${COMMITS}" -eq 1 ]]; then
+  echo "--------------------------------------------------------------------------------";
+  echo -e "$ONE_COMMIT_MESSAGE";
+  echo "--------------------------------------------------------------------------------";
+  printf "$ONE_COMMIT_MESSAGE" | git commit -F -
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)